src/components/Home/BeforeAndAfter.js

```diff
@@ -38,7 +38,11 @@ export default function BeforeAndAfter() {
                         <AnimateIntoView className="absolute left-20">
                             <img src={factoBlurb} />
                         </AnimateIntoView>
-                        <StaticImage className="max-w-[200px] md:max-w-[300px]" src="../../images/facto-home.png" />
+                        <StaticImage
+                            alt=""
```

Comment:
If we're adding the alt attribute we may as well add some text :) Google will like it for the SEO, and it improves accessibility.

Reply:
Done, good idea!